lib/chefspec/server_runner.rb

```diff
@@ -99,5 +99,24 @@ def port
 
       return @port
     end
+
+    #
+    # The path to cache files on disk.  Unlike the SoloRunner, this path needs
+    # to remain consistent for every Chef run.  Else the Chef client tries to
+    # loads the same cookbook multiple times and will encounter deprecated logic
+    # when creating LWRPs.  This method can be removed when the deprecated logic
+    # is removed.
+    #
+    # The method adds a {Kernel.at_exit} handler to ensure the temporary
+    # directory is deleted when the system exits.
+    #
+    def file_cache_path
+      @@file_cache_path ||= begin
```

Comment:
It does not need to be a class variable.  The only thing I am trying to accomplish is making sure that every instance of `ServerRunner` sets the same value in `Chef::Config[:file_cache_path]`.

Do you have a suggestion for the right way to do this?  I can add in a mutex and lock on that if you think thats the best answer.